### ISSUE TITLE
Hide information on connection string

### ISSUE THREAD

**Issue by bbolek** [aws/aws-xray-sdk-dotnet#167]
Hello;

We can see all details in connection string except password. [Remove Password Line](https://github.com/aws/aws-xray-sdk-dotnet/blob/8a9c600ee656a22b2a28bf873d2d069a69c8b9d9/sdk/src/Handlers/EntityFramework/EFInterceptor.cs#L303)

We do not want to display user name and host information. I could not find a way to accomplish this.

Can you provide us a way to hide these information as well?

Thanks

**Comment by Encilo** [comment]
Yes it would be really good to have control over this since other parts of the connection string are sensitive as well.

Thanks

**Comment by lupengamzn** [comment]
Hi @bbolek @Encilo ,

Thanks for the feedback

This is indeed a necessary feature that we need to pay attention to. As this occurs across all X-Ray SDKs, we would need to figure out a way to control the recording of some of the confidential information while tracing SQL queries. I will put this issue into our backlog and hopefully we can address it soon.
